SPEC.md

```diff
@@ -93,25 +94,33 @@ foo 1 key="val" 3 {
 ### Identifier
```

Comment:
Do we want to rephrase the ident/string concepts so idents are a type of String, rather than strings being a kind of ident? That mental change is what finally flipped me on the `#true` change.

Reply:
I think I really like this idea, and I was definitely feeling something along these lines might help it be more clear that they're just a different type of string as far as the language is concerned.

Reply:
This is node!